query-engine/connector-test-kit-rs/query-tests-setup/src/connector_tag/mongodb.rs

```diff
@@ -148,6 +148,5 @@ impl ToString for MongoDbVersion {
 }
 
 fn mongo_capabilities() -> Vec<ConnectorCapability> {
-    let dm_connector = MongoDbDatamodelConnector::default();
-    dm_connector.capabilities().to_owned()
+    MongoDbDatamodelConnector.capabilities().to_owned()
```

Comment:
I'd maybe also have `&'static [ConnectorCapability]` here, but it's already on the QE territory...

Reply:
You could change it, but it's not worth the effort. You'd need to touch the macros.